tests/framework/CodeInterfaceTests/tests

```diff
@@ -139,6 +139,13 @@
    csv = 'PrescientInterfaceTest/out_prescient_fake_0.csv PrescientInterfaceTest/out_prescient_fake_1.csv'
    text = 'PrescientInterfaceTest/runPrescientFake/1/deterministic_with_network_scenarios_day/pyspdir_twostage/2020-07-10/Scenario_actuals.dat PrescientInterfaceTest/runPrescientFake/2/deterministic_with_network_scenarios_day/pyspdir_twostage/2020-07-10/Scenario_actuals.dat '
    test_interface_only = True
-   
  [../]
+
+ [./s_whitmoreTest]
```

Comment:
This could use a name that reflects that this tests abce.

Reply:
Good point. Let me know if you have any suggestions or ideas.

Reply:
I've named it abceTest for now.